test/libsolidity/SolidityNatspecJSON.cpp

```diff
@@ -485,6 +485,141 @@ BOOST_AUTO_TEST_CASE(event)
 	checkNatspec(sourceCode, "ERC20", userDoc, true);
 }
 
+BOOST_AUTO_TEST_CASE(emit_event_from_another_contract)
+{
+	char const* sourceCode = R"(
+		contract X {
+			/// @notice E event
+			/// @dev E event
```

Comment:
The test would be a little more effective if these messages where different. As is, it cannot detect them being swapped.

```suggestion
			/// @notice Userdoc for event E.
			/// @dev Devdoc for event E.
```